### ISSUE TITLE
partitions.csv not being picked up.

### ISSUE THREAD

**Issue by axos88** [ivmarkov/rust-esp32-std-demo#116]
sdkconfig.defaults is missing:

```
CONFIG_PARTITION_TABLE_SINGLE_APP=
CONFIG_PARTITION_TABLE_CUSTOM=y
```

But even then the build is looking for the partitions.csv in the wrong place

**Comment by axos88** [comment]
I needed to add the following as well:

```
CONFIG_PARTITION_TABLE_CUSTOM_FILENAME="../../../../../../partitions.csv"
```

**Comment by pyaillet** [comment]
`espflash` documentation [here](https://github.com/esp-rs/espflash/blob/master/cargo-espflash/README.md#package-metadata) states that you can either pass this as a parameter to the command or add a configuration in `Cargo.toml`

**Comment by axos88** [comment]
The problem is that the example project has this file in the root directory, but it's not actually used, which is misleading. Either it would need to be added to the README.md, or these config options need to be added, so a newcomer is not confused by their edits to the partition table file not being picked up.

**Comment by pyaillet** [comment]
Yes, that's why I proposed a PR #117 adding the configuration to `Cargo.toml` :)

**Comment by axos88** [comment]
Sorry, I did not see that!
